yammbs/_base/base.py

```diff
@@ -1,15 +1,11 @@
 """Copied from openff-nagl."""
```

Comment:
(very minor) Do you want to update/delete this one too like in array.py?

Reply:
Good catch, I think I was reading too fast when I made the other change. I actually want to add the header and keep the attribution in place, though it's probably not a big deal given the VCS history